app/javascript/src/autosave.js

```diff
@@ -0,0 +1,7 @@
+$(document).on("change", ".edit_patient", () => {
```

Comment:
Can you wrap this whole thing in `$(document).on('turbolinks:load', => {...}`? This is basically the turbolinks version of `$(document).ready` or whatever it is -- we want these handlers to attach to things once we're sure they've loaded.

```suggestion
$(document).on("turbolinks:load", () => {
  $(document).on("change", ".edit_patient", () => {
```

Reply:
I think I captured what you are looking for in the latest commit.